charts/observability-pipelines-worker/templates/_pod.tpl

```diff
@@ -80,7 +84,7 @@ containers:
 {{- end }}
     volumeMounts:
       - name: data
-        mountPath: "{{ .Values.config.data_dir | default "/var/lib/observability-pipelines-worker" }}"
+        mountPath: "{{ .Values.datadog.data_dir | default "/var/lib/observability-pipelines-worker" }}"
```

Comment:
Curious how OPW handles things if a `data_dir` is specified in both the pipelines config and the bootstrap/opw side.

Reply:
As of https://github.com/DataDog/observability-pipelines-worker/pull/81, if `data_dir` is set in the pipeline config, we'll raise a validation error and OPW will fail to start e.g.
```
Configuration is invalid.
-------------------------
x Pipeline configuration must not contain data_dir setting. This should be set in the bootstrap configuration.
```

Reply:
Ah, that sounds familiar - just wanted to make sure that's covered.